assets/templates/buttons-component.html

```diff
@@ -371,6 +371,143 @@ <h3>
+                      </div>
+
+                      <div class="dp-text-switch m-b-12">
+                        <div class="dp-switch">
+                          <input type="checkbox" id="threeCheck" />
+                          <label for="threeCheck">
+                            <span></span>
+                          </label>
+                        </div>
+                        <p>
+                          Sólo envíos en franja horaria (tener en cuenta el
+                          horario del país receptor del SMS)
+                        </p>
+                      </div>
+
+                      <div class="dp-text-switch m-b-12">
+                        <div class="dp-switch">
+                          <input type="checkbox" id="fourCheck" checked />
+                          <label for="fourCheck">
+                            <span></span>
+                          </label>
+                        </div>
+                        <p>Sólo envíos de Lunes a Viernes</p>
+                      </div>
+
+                      <div class="dp-text-switch m-b-12">
+                        <div class="dp-switch">
+                          <input type="checkbox" id="fiveCheck" />
+                          <label for="fiveCheck">
+                            <span></span>
+                          </label>
+                        </div>
+                        <p>Sólo envíos de Lunes a Viernes</p>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+
+                <div class="col-sm-12 col-md-8 col-lg-6 m-b-12">
+                  <div class="dp-box-shadow dpsg-content-sample">
+                    <div class="buttons-card">
+                      <div class="button-description">
+                        <p class="m-b-18"><strong>Slide</strong></p>
+                      </div>
+
+                      <div class="dp-calc-slider progress-bar">
+                        <input
+                          class="range-slider"
+                          type="range"
+                          min="0"
+                          value="0"
+                          max="5"
+                          step="1"
+                          data-value="0"
+                        />
+                        <div class="progress-anchor" style="width: 30%"></div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+
+                <div class="col-sm-12 col-md-8 col-lg-6 m-b-12">
+                  <div class="dp-box-shadow dpsg-content-sample">
+                    <div class="buttons-card">
+                      <div class="button-description">
+                        <p class="m-b-18"><strong>Slide Double</strong></p>
+                      </div>
+
+                      <!-- Range Slider HTML -->
+                      <div slider id="slider-distance m-b-24">
```

Comment:
The id cannot have a space in the middle, did you mean something like this?
```suggestion
                      <div slider id="slider-distance" class="m-b-24">
```